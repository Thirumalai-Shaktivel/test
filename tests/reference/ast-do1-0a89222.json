{
    "basename": "ast-do1-0a89222",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/do1.f90",
    "infile_hash": "8a141fb043e826435233a5b5dd627eb5572956821fc11837b4f2707c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-do1-0a89222.stdout",
    "stdout_hash": "0be36a357a4ad5f0b9272987c166e715051c07e91c131e7798db8748",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}