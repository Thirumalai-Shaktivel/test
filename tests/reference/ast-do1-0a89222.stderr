warning: Style suggestion: write 'end do' instead of 'enddo'
  --> tests/do1.f90:21:1
   |
21 | enddo
   | ^^^^^ help: write it as 'end do' to silence this warning
